ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/Committer.scala

```diff
@@ -83,7 +87,7 @@ private[kvutils] trait Committer[Submission, PartialResult] {
         result match {
           case StepContinue(newCState) => cstate = newCState
           case StepStop(logEntry) =>
-            return logEntry -> ctx.getOutputs.toMap
+            return logEntry -> SortedMap(ctx.getOutputs.toSeq: _*)
```

Comment:
This is nice, but I think we should do it in a separate change because I can't quite follow the ramifications.

Reply:
Pulled it out and created #5013.